src/main/java/com/ays/admin_user/controller/AdminUserRegisterApplicationController.java

```diff
@@ -131,4 +132,20 @@ public AysResponse<AdminUserRegisterApplicationCreateResponse> createRegistratio
         );
     }
 
+    /**
+     * Rejects an admin user register application.
+     * Requires SUPER_ADMIN authority.
+     *
+     * @param id The id of the register application.
+     * @return A response object containing the rejected register application.
+     */
+    @PostMapping("/registration-application/{id}/reject")
+    @PreAuthorize("hasAnyAuthority('SUPER_ADMIN')")
+    public AysResponse<Void> rejectRegistrationApplication(
+            @PathVariable @UUID String id, @RequestBody @Valid AdminUserRegisterApplicationRejectRequest request) {
```

Comment:
Aşağıdaki şekilde daha düzenli durabilir
```suggestion
    public AysResponse<Void> rejectRegistrationApplication(@PathVariable @UUID String id, 
                                                           @RequestBody @Valid AdminUserRegisterApplicationRejectRequest request) {
```

Reply:
Düzeltildi